### ISSUE TITLE
Hold CV - Feature

### ISSUE THREAD

**Issue by blackearth2014** [mschack/VCV-Rack-Plugins#33]
What is the "Hold CV" feature?

**Comment by blackearth2014** [comment]
Never mind. It ignores the gate information for each pattern when using the 3 pitched knobs. It sounds really smooth. Thanks.

**Comment by mschack** [comment]
Yeah, I couldn't figure out why my bass drum synth was cutting off when I cranked up the release.  Looked through the code in vain, then finally realized the sequencer pattern cut it off when a pattern value wasn't set.

**Comment by blackearth2014** [comment]
The 3OSC is sounding better too. When I use Bogaudio envelope generators to modulate the filters, its all butter. Even when I am pushing the Res to max and lowering the Filter with the modulation sweeps, there is not aliasing or breaking of the sound. I don't know the dsp bottom end you put in the 3OSC, but but earphones are thankful for the low-end rumble.

For random melodic sequencing, do you plan to add a MSHack S&H special?

**Comment by mschack** [comment]
I am currently working on an ARP.  of sorts...

**Comment by mschack** [comment]
A quicky mock-up
![image](https://user-images.githubusercontent.com/32986033/33791091-eea48a7c-dc54-11e7-947d-79e0a948d975.png)

**Comment by mschack** [comment]
for each note you can program 3 stages, note on off or a rest

**Comment by mschack** [comment]
modes will be up, down, up/down, up/hold/down, down/hold/up, probably will add some sort of spiral mode and random too

**Comment by mschack** [comment]
I am also going to legato and glide for each note

**Comment by blackearth2014** [comment]
PERFECTA, PERFECTA, PERFECTA, PERFECTA, PERFECTA, PERFECTA, PERFECTA, PERFECTA, PERFECTA, PERFECTA, 

Can't Wait
